app/views/shared/my_facilities/_facilities_celebration.html.erb

```diff
@@ -0,0 +1,31 @@
+<!-- celebration card -- dont forget to remove images once deleting-->
+<% if CountryConfig.current[:name] == "India" %>
```

Comment:
Hi @harimohanraj89, do you often set things to production only?
I worry it might get forgotten about if it does not show in the dev env, as I imagine we will be removing this in a few weeks time.

Reply:
I also worry about this @jamiecarter7 . But let's set a reminder to ourselves to remove. Hari, what's the logic of what belongs in Prod vs. any environment?

Reply:
Yeah, I'd rely on reminders/stories to remember to remove this kind of stuff.

India prod, Bangladesh prod, Sri Lanka prod, and Ethiopia prod are the "production" environments. Others are "demo", "sandbox" or "qa. My suggestion was to avoid showing the 20 lakh notice in India Demo specifically, as that might be confusing. Not a big deal either way (and actually easier to test if we don't add my suggestion). Let's leave it as is.